fetcher/fetcher_handlers.go

```diff
@@ -192,6 +192,11 @@ func (c *Fetcher) fetchServiceBindings(session *SessionExt, entry *models.CFObje
 	if err == nil {
 		loadIndex(entry.ServiceBindings, bindings, func(r resources.ServiceCredentialBinding) string { return r.GUID })
 	}
+
+	routeBindings, _, _, err := session.V3().GetRouteBindings(LargeQuery)
```

Comment:
I think fetch should go through its own fetch function `fetchServiceRouteBinding` in order to take maximum advantage of the parallelization offered by the worker system.